features/buyers/accounts/new.feature

```diff
@@ -37,3 +37,22 @@ Feature: Audience > Accounts > New
     And field "Email" has inline error "should look like an email address"
     And field "Organization/Group Name" has inline error "can't be blank"
     But field "Password" has no inline error
+
+  @javascript
+  Scenario: Create account with fields with choices
+    Given the provider has the following fields defined for accounts:
+      | name                 | choices                 | label                  | required | read_only | hidden |
```

Comment:
I suggest using upper case to highlight the table header and also to match the labels of the Field definition form:
```suggestion
      | Name                 | Choices                 | Label                  | Required | Read only | Hidden |
```